src/components/Tree/Node.tsx

```diff
@@ -141,27 +135,33 @@ export const Node = ({
             >
                 <div
                     className={merge([
```

Comment:
Some styling tokens weren't the correct ones according to design. I made some suggestions.

Reply:
Good catch, updated! Thanks!